src/UI/Conversation.jsx

```diff
@@ -353,6 +374,10 @@ class Conversation extends Component {
 							</React.Fragment>
 						))
 					}
+					{
+						this.welcomeMessageAnnouncementRendered === false
+						&& this.renderWelcomeMessageAnnouncement()
```

Comment:
Deze check zit ook in the render functie

```suggestion
						this.renderWelcomeMessageAnnouncement()
```

Reply:
Oops, fixed 8300965eb58543e287fff34b4592394419d26264